src/Microsoft.AspNetCore.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -65,7 +64,7 @@ public DateHeaderValues GetDateHeaderValues()
 
             return new DateHeaderValues()
             {
-                Bytes = _activeDateBytes ? _dateBytes0 : _dateBytes1,
+                Bytes = _dateBytes,
```

Comment:
Re less work in `GetDateHeaderValues` could class struct it to copy rather than create? Not sure costs https://github.com/benaadams/KestrelHttpServer/commit/3a992bd55b2a56e9e8ab81a4dabbf31b579e217a